src/main.rs

```diff
@@ -139,6 +144,19 @@ fn main() {
                 .required(false)
                 .requires("config-file")
         )
+        .arg(
+            Arg::with_name("logs-path")
+                .long("logs-path")
+                .takes_value(true)
+                .required(false)
+                .requires("metrics-path")
+        )
+        .arg(
+            Arg::with_name("metrics-path")
+                .long("metrics-path")
+                .takes_value(true)
+                .required(false)
```

Comment:
should also have `.requires("logs-path")` here since the both fields are mandatory for the logger config.

Reply:
True, nice catch! Don't know why I was under the impression that `requires` behaves as `conflicts_with` and I can put `.requires` only for one of the 2 arguments.